etor/src/main/java/gov/hhs/cdc/trustedintermediary/etor/utils/security/HashHelper.java

```diff
@@ -0,0 +1,26 @@
+package gov.hhs.cdc.trustedintermediary.etor.utils.security;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import java.util.HexFormat;
+
+public class HashHelper {
+
+    private static final HashHelper INSTANCE = new HashHelper();
+
+    public static HashHelper getInstance() {
```

Comment:
Mind making a follow-up PR to add a private constructor?  I'm happy to collaborate with you on that.